internal/provider/resource_privatelink.go

```diff
@@ -37,6 +39,7 @@ type PrivateLinkModel struct {
 	ConnectionName types.String `tfsdk:"connection_name"`
 	Target         types.String `tfsdk:"target"`
 	Endpoint       types.String `tfsdk:"endpoint"`
+	WaitForReady   types.Bool   `tfsdk:"wait_for_ready"`
```

Comment:
I don't see this variable being used in the PR, is this expected?

Reply:
yes, we currently do not have vpc endpoints for testing.

Reply:
Not just for testing, I didn't see any reference to this variable in the PR including the implementation. Did I miss anything?

Reply:
I forgot to commit the change